pontoon/base/models.py

```diff
@@ -360,6 +360,17 @@ def serialized_notifications(self):
     }
 
 
+def user_serialize(self):
+    """ Serialize Project contact """
```

Comment:
I set this up to send all the needed data for a mention, but ATM I'm only using the `name`